### PR TITLE
Inline edit width

### PR DESCRIPTION
**Pull request by jenzweben**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/ilios/frontend/pull/196" target="_blank">ReviewNinja

File: app/styles/components/_global.scss [hunk 1]
```diff
@@ -15,16 +15,24 @@ button {
...
   vertical-align: middle;
   white-space: nowrap;
 }
 
+h2 {
+  font-size: 20px;
+}
+h4 {
+  font-size: 16px;
+  color: #666;
```
Comment (jrjohnson) [inline app/styles/components/_global.scss]: Colors should be in variables scss.

File: app/styles/components/_detailview.scss [hunk 2]
```diff
@@ -120,7 +123,7 @@
 
   .detail-header {
     @include span-columns(12 of 12);
-    background-color: $background-grey;
+    background-color:#edecec;
```
Comment (jrjohnson) [inline app/styles/components/_detailview.scss]: colors should be variables.

**Comment by jrjohnson** [comment]
@jenzweben - this is still waiting for some minor edits.

**Pull request merged**